compiler/plugins/target/AMD-AIE/aievec/VectorToVectorConversions.cpp

```diff
@@ -303,6 +304,92 @@ class FlattenContiguousRowMajorTransferWritePattern
 
 }  // namespace copied_from_mlir
 
+/// Utility to check if the indices provided are all 0.
+static LogicalResult isAllZeroOffsetAccess(mlir::OperandRange indices) {
+  if (!llvm::all_of(indices, [](Value val) {
+        IntegerAttr attr;
+        if (!matchPattern(val, m_Constant(&attr))) return false;
+        return attr.getInt() == 0;
+      })) {
+    return failure();
+  }
+  return success();
+}
+
+/// Utility to fetch indices of Subview op which would be used by a new vector
+/// transfer_read/transfer_write op with trivial access pattern.
```

Comment:
This function seems to just convert from OpFoldResults to Values, I think the comment should be about what it does not how it is used. Similarly I think it should be named `opFoldResultsToValues`